.github/workflows/e2e-test-tf-module.yml

```diff
@@ -62,16 +56,30 @@ jobs:
         with:
           ref: ${{ inputs.ref || github.head_ref }}
 
-      - name: Upload module
+      - name: Find Latest Image
```

Comment:
nit: Get latest image (also for ID to suggest that it might not need to find it)